hoot-services/src/main/java/hoot/services/controllers/ingest/CustomScriptResource.java

```diff
@@ -1001,12 +1001,12 @@ public static Translations getTranslationForUser(Users user, Long translationId)
         }
 
         if(user != null && !isVisible(user, translationId)) {
-            throw new ForbiddenException(Response.status(Response.Status.FORBIDDEN).type(MediaType.TEXT_PLAIN).entity("You do not have access to this translation").build());
+            throw new ForbiddenException("You do not have access to this translation");
         }
 
         // Check if owner of translation isn't the user, user isn't admin, and there isn't an owner of the translation
         if(user != null && !translation.getUserId().equals(user.getId()) && !UserResource.adminUserCheck(user) && translation.getUserId() != -1) {
-            throw new ForbiddenException(Response.status(Response.Status.FORBIDDEN).type(MediaType.TEXT_PLAIN).entity("You must own the translation to modify it").build());
+            throw new ForbiddenException("You do not have access to this translation");
```

Comment:
I still think this one should be "You must own the translation to modify it"
access implies the user can't see or read it